src/content/docs/apm/agents/ruby-agent/getting-started/migration-10x-guide.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Ruby agent 9.x to 10.x migration guide
+tags:
+  - Agents
+  - Ruby agent
+  - Getting started
+redirects:
+  - /docs/agents/ruby-agent/getting-started/migration-10x-guide
+freshnessValidatedDate: never
+---
+
+This guide covers the major changes between the 9.x and 10.x series of the Ruby agent and how to have a successful migration.
+
+## Summary [#summary]
+
+Main changes in v10.0 include:
```

Comment:
We might want to structure this closer to what's at the bottom of the [confluence doc (internal)](https://newrelic.atlassian.net/wiki/spaces/INST/pages/3955949842/APM+Agents+Major+Version+Release+Process) about major version releases.